src/main/java/org/folio/circulation/domain/notice/session/PatronActionSessionProperties.java

```diff
@@ -0,0 +1,12 @@
+package org.folio.circulation.domain.notice.session;
+
+public class PatronActionSessionProperties {
+
```

Comment:
Personally, I would remove this blank line as I don't think it increases the readability of this class

Reply:
Done